### PR TITLE
Cleanup

### PR DESCRIPTION
**Pull request by thomasleese**
Various small things which together clean up the code base.

### CODE REVIEW

File: robotd/devices.py [hunk 1]
```diff
@@ -100,10 +100,14 @@ def name(cls, node):
         """Simple node name."""
         return node.sys_name
 
+    def read_temperature_value(self):
+        path = '{}/temp'.format(self.node.sys_path)
+        with open(path) as file:
```
Comment (Adimote) [inline robotd/devices.py:105]: (nitpick) personally I wouldn't override the `file` keyword here

**Review by Adimote — CHANGES_REQUESTED** [review]
for cfb66f8 can you also fix the double -> single quotes for the strings in `devices.py`?

File: robotd/devices.py (hunk 1 above)
Comment (RealOrangeOne) [reply robotd/devices.py:105]: FTR `f` is the standard shorthand shorthand convention here

File: robotd/usb.py [hunk 2]
```diff
@@ -95,6 +98,8 @@ def control_read(self, request, value, index, length, timeout=3000):
 
 
 def enumerate():
```
Comment (RealOrangeOne) [inline robotd/usb.py]: Not really related to your change, but could this function be renamed (`enumerate_devices`), I dont like overriding builtin functions like this

File: robotd/devices.py (hunk 1 above)
Comment (thomasleese) [reply robotd/devices.py:105]: ‘file’ no longer exists in python 3

File: robotd/usb.py (hunk 2 above)
Comment (thomasleese) [reply robotd/usb.py]: Agreed, but would also agree not related to this PR :P

Comment (RealOrangeOne) [reply robotd/usb.py]: Fancy doing it anyway? ;)

File: robotd/devices.py (hunk 1 above)
Comment (Adimote) [reply robotd/devices.py:105]: Ah I didn't know. never mind then :)

**Review by Adimote — DISMISSED** [review]

File: robotd/usb.py (hunk 2 above)
Comment (thomasleese) [reply robotd/usb.py]: Done!

**Comment by thomasleese** [comment]
@Adimote @RealOrangeOne Comments addressed :)

**Review by RealOrangeOne — APPROVED** [review]

**Pull request merged**